packit/base_git.py

```diff
@@ -275,3 +275,13 @@ def add_patches_to_specfile(self, patch_list: List[Tuple[str, str]]) -> None:
         )
         self._specfile = None  # reload the specfile object
         self.local_project.git_repo.index.write()
+
+    def get_project_url_from_distgit_spec(self) -> Optional[str]:
+        sections = self.specfile.spec_content.sections
+        package_section: List[str] = sections.get("%package", [])
+        for s in package_section:
+            if s.startswith("URL:"):
+                url = s[4:].strip()
+                logger.debug(f"Upstream project URL = {url}")
+                return url
+        return None
```

Comment:
Maybe, you can utilize a `rebasehelper.Specfile`  more. Just look at [the `download_remote_sources` method](https://rebase-helper.readthedocs.io/en/latest/_modules/rebasehelper/specfile.html#SpecFile.download_remote_sources).

But this should work good as well.

Reply:
I added a note to the code.